src/api/routes/rosetta/account.ts

```diff
@@ -122,17 +121,10 @@ export function createRosettaAccountRouter(db: DataStore, chainId: ChainID): Rou
   return router;
 }
 
-function getVestingInfo(info: TokenOfferingLocked, block_height: number): any {
+function getVestingInfo(info: AddressTokenOfferingLocked): any {
```

Comment:
Unnecessary `any` usage -- can you specify the type here?

Reply:
Done!